### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by sakin97** [ynct-d-members/todo-team#26] [Ideas]
API設計と実装，クライアント側での呼び出しについて扱うスレッド

**Comment by sakin97** [comment]
アイデア: 以下を参考にaxiosでの呼び出し処理を共通化する
https://qiita.com/taskeknight/items/d5bc90928d174993c6d1

**Reply by sakin97** [reply]
axiosのカスタムhookをライブラリ化してるものがあった
https://github.com/use-hooks/react-hooks-axios

**Reply by shippokun** [reply]
あんまりメンテされてないから使うのは微妙かな
